chromium/browser_type.go

```diff
@@ -179,8 +179,9 @@ func (b *BrowserType) launch(
 	if err != nil {
 		return nil, 0, fmt.Errorf("%w", err)
 	}
+
 	dataDir := &storage.Dir{}
-	if err := dataDir.Make("", flags["user-data-dir"]); err != nil {
+	if err := dataDir.Make(b.tmpdir(), flags["user-data-dir"]); err != nil {
```

Comment:
Was this a bug? Seems like we weren't working with `TEMPDIR` at until this change 🙂

Reply:
Most probably 😅